experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -309,4 +326,61 @@ export class NodeSDK {
         .then(() => {})
     );
   }
+
+  private configureLoggerProviderFromEnv(): void {
+    const logExportersList = process.env.OTEL_LOGS_EXPORTER;
+    if (!logExportersList) {
+      diag.info(`No log exporters specified. Logs will not be exported.`);
+      return;
+    }
+
+    const exporters: LogRecordExporter[] = [];
+    const enabledExporters = filterBlanksAndNulls(logExportersList.split(','));
+
+    if (enabledExporters.includes('none')) {
+      diag.info(
+        `OTEL_LOGS_EXPORTER contains "none". Logger provider will not be initialized.`
+      );
+      return;
+    }
+
+    enabledExporters.forEach(exporter => {
+      if (exporter === 'otlp') {
+        const protocol = (
+          process.env.OTEL_EXPORTER_OTLP_LOGS_PROTOCOL ??
+          process.env.OTEL_EXPORTER_OTLP_PROTOCOL
+        )?.trim();
+        switch (protocol) {
+          case 'grpc':
+            exporters.push(new OTLPGrpcLogExporter());
+            break;
+          case 'http/json':
+            exporters.push(new OTLPHttpLogExporter());
+            break;
+          case 'http/protobuf':
+            exporters.push(new OTLPProtoLogExporter());
+            break;
+          default:
+            diag.debug(
+              `Unsupported or undefined OTLP logs protocol. Using http/protobuf.`
+            );
+            exporters.push(new OTLPProtoLogExporter());
+        }
+      } else if (exporter === 'console') {
+        exporters.push(new ConsoleLogRecordExporter());
+      } else {
+        diag.warn(
+          `Unsupported log exporter: ${exporter}. Supported exporters are: otlp, console.`
+        );
+      }
+    });
+
+    if (exporters.length > 0) {
+      this._loggerProviderConfig = {
+        logRecordProcessors: exporters.map(
+          exporter => new BatchLogRecordProcessor(exporter)
```

Comment:
As with SpanProcessors at https://github.com/open-telemetry/opentelemetry-js/blob/a037f84b679386daefd726133be6265c3c43de27/experimental/packages/opentelemetry-sdk-node/src/TracerProviderWithEnvExporter.ts#L170-L175 perhaps this should avoid the `BatchLogRecordProcessor` when the exporter is the `ConsoleLogRecordExporter`.